app/assets/javascripts/discourse/components/topic-navigation.js.es6

```diff
@@ -109,6 +111,55 @@ export default Ember.Component.extend({
     }
   },
 
+  _panOpenClose(offset, velocity, direction) {
+
+    const maxOffset = parseInt($(".timeline-container").css("height"));
```

Comment:
given you fetch `$(".timeline-container")` multiple times here, I would store it in a variable.

```
$timelineContainer = $(".timeline-container");
```